app/helpers/patient_query_helper.rb

```diff
@@ -241,9 +241,9 @@ def advanced_filter(patients, filters, tz_offset)
                    end
       when 'responded-today'
         patients = if filter[:value].present?
-                     patients.where('latest_assessment_at >= ?', 24.hours.ago)
+                     patients.submitted_assessment_today
```

Comment:
I don't think this scope fits the advanced filter here. Submitting an assessment today is different from submitting an assessment in the last 24 hours, and this would be changing functionality. I'd ask that this be reverted. In which case you likely do not need the `has_not_submitted_assessment_today` scope either.

Reply:
reverting and removing `has_not_submitted_assessment_today`

Reply:
Wanna inform @Bialogs of this change because I see he originally called out the advanced filter above:
> We will be performing different behavior between a search and the new scope for submitted today used when closing patients. For example if a user wanted to see what patients were going to be closed today using the advanced searching they would arrive at a different set because the calculation done in the query helper is not correct based on this new notion of "Today" not being within 24 hours but actually today's date.

@Bialogs Removed because these are two separate things. Advanced Filter is explicitly 24 hours. On the frontend we tell the user "24 hours" rather than "Today"